engine/core/src/test/java/org/datacleaner/reference/regexswap/RegexSwapClientTest.java

```diff
@@ -32,6 +27,8 @@
 import org.junit.Assume;
 import org.junit.Test;
 
+import static org.junit.Assert.*;
```

Comment:
Why remove the four specific imports from above and replace them with a generic import?